.github/workflows/markdown-lint.yml

```diff
@@ -47,10 +68,10 @@ jobs:
         if: ${{ !cancelled() }}
         run: |
           git ls-files --exclude-standard -z -- *.md **/*.md |\
-          xargs -0 -I{} markdownlint "{}" --config .markdownlint.yaml ;
+          xargs -0 -I{} npm run lint:md -- "{}" --config .markdownlint.yaml ;
```

Comment:
@coderabbitai,
This step seems to have regressed ever since we introduced the changes from GHI #214 namely the part where we use `npm run` functions.

Take a look at [the logs](https://github.com/reactive-firewall/multicast/actions/runs/12125827410/job/33806613972?pr=227)

they seem to complain about a shell substitution.

* also consider the changes in the file `pakage.json` (also introduces as part of GHI #209 at the same time) where the npm scripts config is.

Besides forcing the GHA step to use the bash shell, what could be causing the issue when changing to using `npm run`?

because I'm new to npm I'm unsure if this is a shell issue rooted in the GHA itself or in the NPM script.

point out any obvious issue with the "Lint YAML config for markdown" step here.

Reply:
🤔 Looks like 7eb54e2 finally resolved the bug. 🤷🏻